deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
With this change will I be able to run `docker build -t ..` without requiring `make` and `golang`?